### PR TITLE
Duplicate array key

### PR DESCRIPTION
**Pull request by demis-palma**
Removed duplicate array pair key / value.
To be merged on review.

### CODE REVIEW

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on b49921fcf08a5d0e991c05f315536e9d1527c979<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15193">issues.joomla.org/tracker/joomla-cms/15193</a>.</sub>

File: libraries/joomla/form/fields/password.php [hunk 1]
```diff
@@ -169,8 +169,7 @@ protected function getLayoutData()
 		$extraData = array(
 			'maxLength' => $this->maxLength,
 			'meter'     => $this->meter,
-			'threshold' => $this->threshold,
-			'meter'     => $this->meter,
+			'threshold' => $this->threshold
```
Comment (Bakual) [inline libraries/joomla/form/fields/password.php]: Can you add back the comma after the last array member? It plays nicer with Git if there is one.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 7b9604a98d32084370d8b5616cb5930562938893<br /><br />on review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15193">issues.joomla.org/tracker/joomla-cms/15193</a>.</sub>

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 7b9604a98d32084370d8b5616cb5930562938893<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15193">issues.joomla.org/tracker/joomla-cms/15193</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15193">issues.joomla.org/tracker/joomla-cms/15193</a>.</sub>

**Pull request merged**